github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -161,6 +161,42 @@ presubmits:
           path: /sys/fs/cgroup
           type: Directory
         name: cgroup
+  - always_run: false
+    cluster: prow-arm64-workloads
+    decorate: true
+    decoration_config:
+      timeout: 3h0m0s
+    max_concurrency: 1
+    name: check-up-kind-1.33-arm64
+    optional: true
```

Comment:
Does it make sense to make this required?

Reply:
Yeah so if `run_if_changed` is there and optional is not set - it defaults to required. I think it makes sense to move the kind providers to required - if changes are made to them. 

I can handle this in a follow up though. Its not really needed here.